wiki/Bazel-Setup-Instructions-for-Mac.md

```diff
@@ -0,0 +1,105 @@
+Follow the instructions [here](https://installvirtual.com/install-openjdk-8-on-mac-using-brew-adoptopenjdk/) to install OpenJDK 8. 
+Note that this requires the installation of brew as a pre-requisite, which can be done by following the instructions [here](https://mac.install.guide/homebrew/index.html). You can then set up your `$JAVA_HOME` environment variable using these [instructions](https://stackoverflow.com/a/75167958/11396524).
+
+
+### 4. Install Python 2 and make sure it is active in your environment
+
+To install Python 2 in MacOS follow the follows the commands given below. Note that this requires installation of brew as a pre-requisite, which can be done by following the instructions [here](https://mac.install.guide/homebrew/index.html). 
+```
+brew install pyenv
+pyenv install 2.7.18
+pyenv global 2.7.18
+```
+
+- To make sure Python 2 is successfully installed and active in your environment, navigate to the **oppia-android** directory and run the following commands:
+
+```
+export PATH="$(pyenv root)/shims:${PATH}"
+python --version
+```
+
+### 5. Set Bazel, Python 2, ANDROID_HOME paths permanently in your terminal
+
+- To set the `Bazel`, `Python 2`, `ANDROID_HOME` path permanently in your terminal run these commands:
+    ```
+    sudo nano /etc/paths
+    ```
+   - Enter your password, when prompted.
+   - Go to the bottom of the file, and enter these paths 
+     ```
+     /Users/{YourMacUserName}/bin
+     $(pyenv root)/shims:${PATH}
+     $HOME/Library/Android/sdk
+     ```
+   - Hit control-x to quit.
+   - Enter “Y” to save the modified buffer.
+   - That’s it!  To test it, in a new terminal window, type: `echo $PATH`
+
+**Note: You must set the path for `Bazel`, `Python 2`, `ANDROID_HOME` before running bazel build for oppia-android, otherwise you will get an error.**
+
+### 6. Prepare the build environment
+
+Follow the instructions in [oppia-bazel-tools](https://github.com/oppia/oppia-bazel-tools#readme), in order to prepare your environment to support Oppia Android builds.
+
+### 7. Verify that the build is working
+
+At this point, your system should be able to build Oppia Android. To verify, try building the APK (from your subsystem terminal -- note that this and all other Bazel commands must be run from the root of the ‘oppia-android’ directory otherwise they will fail):
+
+```sh
+bazel build //:oppia
+```
+
+(Note that this command may take 10-20 minutes to complete depending on the performance of your machine).
+
+If everything is working, you should see output like the following:
+
+```
+Target //:oppia up-to-date:
+  bazel-bin/oppia_deploy.jar
+  bazel-bin/oppia_unsigned/apk
+  bazel-bin/oppia/apk
+INFO: Elapsed time: ...
+INFO: 1 process...
+INFO: Build completed successfully, ...
+```
+
+If you see the above, you can proceed to use your subsystem for Bazel commands while developing Oppia. If you instead see an error, please [file an issue](https://github.com/oppia/oppia-android/issues/new/choose).
```

Comment:
This bit about the subsystem, isn't that windows specific? I noticed it is also mentioned in the Linux instructions.

Reply:
Done.